### ISSUE TITLE
[Question] MiniGrid Training Problem

### ISSUE THREAD

**Issue by skyler-sky** [DLR-RM/rl-baselines3-zoo#500]
### ❓ Question

I installed stable-baseline3 and rl-baselines3-zoo. I install minigrid（3.0.0） with `pip install minigrid`. 
I do the training with `python train.py --algo ppo --env MiniGrid-Empty-Random-5x5-v0 --eval-freq 10000 --eval-episodes 10 --n-eval-envs 1`
The config are

> MiniGrid-Empty-Random-5x5-v0: &minigrid-defaults
  env_wrapper: minigrid.wrappers.FlatObsWrapper # See GH/1320#issuecomment-1421108191
  normalize: true
  n_envs: 8 # number of environment copies running in parallel
  n_timesteps: !!float 1e5
  policy: 'MlpPolicy'
  n_steps: 128 # batch size is n_steps * n_env
  batch_size: 64 # Number of training minibatches per update
  gae_lambda: 0.95 #  Factor for trade-off of bias vs variance for Generalized Advantage Estimator
  gamma: 0.99
  n_epochs: 10 #  Number of epoch when optimizing the surrogate
  ent_coef: 0.0 # Entropy coefficient for the loss calculation
  learning_rate: 2.5e-4 # The learning rate, it can be a function
  clip_range: 0.2 # Clipping parameter, it can be a function

I dont change anything else. But the training curve is awkward：

<img width="585" height="366" alt="Image" src="https://github.com/user-attachments/assets/c254a664-639a-4288-bf1a-e187a5cbb449" />

Anyone has thoughts why this happen?

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/DLR-RM/rl-baselines3-zoo/issues) in the repo
- [x] I have read the [SB3 documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [x] I have read the [RL Zoo documentation](https://rl-baselines3-zoo.readthedocs.io)
- [x] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [x] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
Reading https://github.com/DLR-RM/stable-baselines3/issues/1320#issuecomment-1421108191, the issue might come from the normalization.

So doing `normalize: false` in the yaml file (`ppo.yaml`) should solve your problem.
Would you mind testing it with the other minigrid envs and doing a PR if it works?

**Comment by NathanGavenski** [comment]
Did you have any luck with that @araffin ? Did you manage to train using `normalize_image=False` ? I did try increasing the budget, but it doesn't seem to resolve the issue.
